Yoast/YoastSEO.js
# Refactor subheadings keyword assessment

## Summary

This PR can be summarized in the following changelog entry:

* Changes the subheadings keyword assessment to also take the morphological forms of the key words into account. Also adds limits to make sure that at least 30%, but no more than 75% of subheadings reflect the topic of an article.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Open a new post in WordPress.
* Enter four words as your keyphrase.
* Enter a text with headers (e.g. `<h1>` or `<h2>`)
* In the header:
      1. Enter two of the four keywords into the header.
          * The header should not be counted in the subheadings keyword assessment.
      2. Enter three of the four keywords into the header.
          * The header should be counted.
      3. Repeat steps 1 and 2, but now for different morphological forms of the keywords (e.g. plural or singular forms). The results should not change. 
* By `counted` the general schema of the assessment should be understood.
  * GOOD if 30-75% of subheadings reflect the topic,
  * GOOD if the only subheading in the text reflects the topic,
  * BAD if less than 30% or more than 75% of subheadings reflect the topic.

Fixes #1756
